lib/commands/create.js

```diff
@@ -98,3 +110,36 @@ module.exports = function (program) {
 function replaceInConfig(regex, replacement) {
   util.replaceInFiles([ './app.js', './package.json' ], regex, replacement);
 }
+
+async function spanWithSpinner () {
```

Comment:
Shouldn't this be spawnWithSpinner? And passing the command and its text to it so you can reuse it for other commands would make sense, I think.

On a way less technical note, I would suggest including "this may take a while" in the message. Just because some users (cough me cough) will still suspect the worst even with a spinner and do better with an affirmative confirmation that yes, this is gonna take a minute and not say much in the meantime.

Reply:
Typo goof.